lib/aqm_reading.rb

```diff
@@ -46,17 +46,12 @@ def latest_reading_recorded_at
 
   def measurements
     key_rows = raw_data['Header'][1..-1]
-    value_rows = raw_data['Footer'][1][1..-1].map { |measurement| extract_value(measurement).to_f }
+    value_rows = raw_data['Footer'][1][1..-1].map { |measurement| extract_value(measurement)&.to_f }
 
     key_rows.zip(value_rows).to_h
   end
 
   def extract_value(string)
-    presence(string.split(' ').first)
-  end
-
-  # Checks for the presence of a reading, returns nil when there is no reading
-  def presence(reading)
-    reading == '-' ? nil : reading
+    string&.split(' ')&.first
```

Comment:
If we change to that Rubocop reports an offence :rotating_light: 

```
Running RuboCop...
Inspecting 22 files
................C.....

Offenses:

lib/aqm_reading.rb:55:5: C: Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.
    string.split(' ').first if string
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

22 files inspected, 1 offense detected
RuboCop failed!
```